config/routes.rb

```diff
@@ -287,16 +287,18 @@
     end
 
     namespace :v2 do
-      resources :accounts, only: [], module: :accounts do
-        resources :reports, only: [:index] do
-          collection do
-            get :summary
-            get :agents
-            get :inboxes
-            get :labels
-            get :teams
-            get :conversations
-            get :conversation_traffic
+      resources :accounts, only: [:create] do
+        scope module: :accounts do
```

Comment:
```
resources :accounts, only: [:create], module: :accounts do
end
```

won't this work ?  again not to impact the routes which are not relevant to this PR